### PR TITLE
Remove whereabouts

### PR DESCRIPTION
**Pull request by sebrandon1**
Testing out what happens if we remove the `whereabouts` repository resources from our CI.

### CODE REVIEW

**Comment by sebrandon1** [comment]
Seems to work just fine.  I'd like others to try this out to see if I'm missing something?

**Review by rdavid — APPROVED** [review]

File: scripts/deploy-multus-network.sh [hunk 1]
```diff
@@ -7,7 +7,6 @@ SCRIPT_DIR=$(dirname "$0")
 source "$SCRIPT_DIR"/init-env.sh
 
 MULTUS_GIT_URL="https://github.com/k8snetworkplumbingwg/multus-cni.git"
-WHEREABOUTS_GIT_URL="https://github.com/k8snetworkplumbingwg/whereabouts"
```
Comment (edcdavid) [inline scripts/deploy-multus-network.sh:10]: Without wherebouts, the cni will assign ip addresses unique only per node. So without Whereabouts,  there will be cases where 2 different interfaces on 2 different nodes have the same IP

**Comment by sebrandon1** [comment]
We are having the same error as: https://github.com/k8snetworkplumbingwg/whereabouts/issues/332

**Comment by sebrandon1** [comment]
Closing in favor of #282

**Pull request closed without merging**